v2/cmd/controller/main.go

```diff
@@ -170,7 +173,18 @@ func makeControllerOptions(log logr.Logger, cfg config.Values) controllers.Optio
 				// TODO: do we need GVK here too?
 				return log.WithValues("namespace", req.Namespace, "name", req.Name)
 			},
+			// These rate limits are used for happy-path backoffs (for example polling async operation IDs for PUT/DELETE)
 			RateLimiter: controllers.NewRateLimiter(1*time.Second, 1*time.Minute),
 		},
+		RequeueIntervalCalculator: interval.NewCalculator(
+			// These rate limits are primarily for ReadyConditionImpactingError's
+			interval.CalculatorParameters{
+				//nolint:gosec // do not want cryptographic randomness here
+				Rand:         rand.New(lockedrand.NewSource(time.Now().UnixNano())),
+				BaseDelay:    1 * time.Second,
```

Comment:
Possibly these should be made configurable? At least the MaxFast/MaxSlow and maybe the 1m (which is sorta "MaxHappyPath"? Thoughts?

Reply:
There's merit in making them configurable, but not enough to block this PR. I'd suggest logging an issue to do that as a followup.

Reply:
I filed #2584